README.md

```diff
@@ -16,18 +16,29 @@ For now, that's just [Django](https://www.djangoproject.com/) and [MongoDB](http
+The point of creating CoGS is to have a solution for the MRDC game server that is as portable, maintainable, and transparent as possible. We learned our lesson; we want CoGS to last for years and to be accessible to people who weren't here when we wrote it.
 
-In case this documentation is insufficient, or there are other problems with this system, you can contact [lpulley](https://lpulley.me/).
+In case this documentation is insufficient, or there are other problems with this system, you can contact [lpulley](https://github.com/lpulley/).
 
 ## Detailed Documentation
 
 ### Running and Stopping
 
-Run `docker-compose up -d` in the root of this repo to start all of the CoGS services in the background, or to update the containers if they are already running.
+Run `docker-compose up -d` in the root of this repo to build images for custom containers and start all of the CoGS containers in the background, or to update the containers if they are already running.
 
-To stop the containers, run `docker-compose stop`. To restart them use `docker-compose start`. To stop the containers and delete them, use `docker-compose down` (note that while this does delete volumes, it does **not** delete host files, so for example the database data in `./data-db` will persist). The `down` subcommand is useful for starting from *almost* scratch, such that only data linked to the host filesystem persists, but every service is completely recycled.
+To stop the containers, run `docker-compose stop`. To restart them use `docker-compose start`. To stop the containers and delete them, use `docker-compose down`. Note that while this does delete volume connections, it does **not** delete host files created by volumes, so for example the database data in `./data-db` will persist; images from `Dockerfile`s will also need to be manually deleted if they are to be rebuilt.
 
-To start with the database from scratch, remove `./data-db` after `docker-compose down`. When you start up again, the database will create a new `./data-db`. Note that if you do this, the database will be missing some vital default models from Django, so you must migrate the models to the database. In Django, this is done with `python manage.py migrate`. **However**, since Django is running inside the `web` service, you must use `docker-compose exec <service> <command>` or `docker exec <container> <command>`. In this case, that would be `docker-compose exec web python manage.py migrate`.
+The `down` subcommand is useful for starting from *almost* scratch, such that only data in volume folders persists, but every container is completely recycled. Running `docker-compose up -d` will start them anew, but does not rebuild images from `Dockerfile`s if they still exist. To rebuild images from `Dockerfile`s use `docker-compose build`. This will force a rebuild of images that are listed in `docker-compose.yml` with a `build:` directory.
```

Comment:
@mgale456 Not sure if you saw my slight change here, but I think you might find `docker-compose build` useful. I didn't know about it before but it does exactly what you're looking for, without having to remove images manually.